tools/python_api/test/test_arrow.py

```diff
@@ -480,6 +480,16 @@ def test_to_arrow_map(conn_db_readonly: ConnDB) -> None:
     ]
 
 
+def test_to_arrow_array(conn_db_readonly: ConnDB) -> None:
```

Comment:
I would also add a test case which exports a mixed NULLs and non-NULLs in the same arrow table.

Reply:
Added